experimenter/experimenter/experiments/models.py

```diff
@@ -875,26 +876,26 @@ def get_changelogs_by_date(self):
                 change = {
                     "event": "GENERAL",
                     "event_message": (
-                        f"{changelog.changed_by.get_full_name()} "
+                        f"{changelog.changed_by} "
                         f"changed value of {field} from "
                         f"{field_diff['old_value']} to {field_diff['new_value']}"
                     ),
-                    "changed_by": changelog.changed_by.get_full_name(),
+                    "changed_by": changelog.changed_by,
                     "timestamp": timestamp,
                 }
 
                 changes_by_date[changelog.changed_on.date()].append(change)
 
         if changelogs:
             creation_log = changelogs[-1]
-            first_timestamp = creation_log.changed_on.strftime(date_option)
+            first_local_timestamp = timezone.localtime(creation_log.changed_on)
+            first_timestamp = first_local_timestamp.strftime(date_option)
             change = {
                 "event": "CREATION",
                 "event_message": (
-                    f"{creation_log.changed_by.get_full_name()} "
-                    f"created this experiment"
+                    f"{creation_log.changed_by} " f"created this experiment"
```

Comment:
Two strings got moved onto the same line, so you can make this a single string:

```suggestion
                    f"{creation_log.changed_by} created this experiment"
```

Reply:
Oh I think it changed after the `make code_format`. I'll update it :+1: